independent-publisher-connectors/Spotify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,943 @@
+                },
+                "next": {
+                  "type": "string",
+                  "x-ms-summary": "Next",
+                  "description": "URL to the next page of items. ( null if none)"
+                },
+                "offset": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Offset",
+                  "description": "The offset of the items returned (as set in the query or by default)"
+                },
+                "previous": {
+                  "type": "string",
+                  "x-ms-summary": "Previous",
+                  "description": "URL to the previous page of items. ( null if none)"
+                },
+                "total": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Total",
+                  "description": "The total number of items available to return."
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get a User's Top Artists",
+        "description": "Get the current user's top artists based on calculated affinity.",
+        "operationId": "GetTopArtists",
+        "parameters": []
+      }
+    },
+    "/v1/browse/new-releases": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "albums": {
+                  "type": "object",
+                  "properties": {
+                    "items": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/definitions/SimplifiedAlbum"
+                      },
+                      "x-ms-summary": "Albums",
+                      "description": "A list of new album releases featured in Spotify."
+                    }
+                  },
+                  "description": "A list of new album releases featured in Spotify (shown, for example, on a Spotify player's 'Browse' tab)."
+                }
+              }
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "country",
+            "x-ms-summary": "Country",
+            "description": "Provide this parameter if you want the list of returned items to be relevant to a particular country. If omitted, the returned items will be relevant to all countries.",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "GetAvailableMarkets",
+              "value-collection": "Markets"
+            }
+          }
+        ],
+        "summary": "Get All New Releases",
+        "description": "Get a list of new album releases featured in Spotify (shown, for example, on a Spotify player's 'Browse' tab).",
```

Comment:
@Laskewitz Can you please improve the description, like below:

`"description": "A list of new album releases featured in Spotify (Example: 'Browse' tab on the Spotify player)."`

Reply:
Fixed.